README.md

```diff
@@ -49,3 +49,7 @@ with any additional questions or comments.
 ## Roadmap
 
 For details on our planned features and future direction please refer to our [roadmap](https://github.com/microsoft/TypeScript/wiki/Roadmap).
+
+## Product Support and EOL
+
+Product support and lifecycles vary based on the patches and new versions that are released. To find more information regarding further releases follow the link here https://devblogs.microsoft.com/typescript/. This blog will keep up-to-date with the latest patches and updates to TypeScript. For further information, please consult the Roadmap.  

```

Comment:
All this says is that the support and lifecycles vary, and where to learn about new releases. Where would one go to learn about the actual support and lifecycles?

Reply:
I agree, I don't think that this provides the requested information in the original issue, and to be honest if we're going to decide on a plan, that's going to come from the core team.